src/addons/MountNode/lib/handleChange.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash'
+import computeClasses from './computeClasses'
+
+let prevClasses = []
```

Comment:
This will be shared across all `MountNode` instances.  If two MountNodes are on the page, they will need their own copy of `prevClasses` to diff against, correct?

This might belong on the actual MountNode class so it can be instance specific.